WordPress/Classes/ViewRelated/System/FancyAlertViewController+SavedPosts.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+import Gridicons
+
+extension FancyAlertViewController {
+    private struct Strings {
+        static let titleText = NSLocalizedString("Save Posts for Later", comment: "Title of alert informing users about the Reader Save for Later feature.")
+        static let bodyText = NSLocalizedString("Save this post, and come back to read it whenever you'd like. It will only be available on this device -- saved posts don't sync to other devices (Yet! We're working on it).", comment: "Body text of alert informing users about the Reader Save for Later feature.")
```

Comment:
I dunno where ` -- ` falls in our style, but I prefer either an unspaced em dash `—` or a spaced en dash ` – `.

Reply:
I'd prefer a proper em dash too – @iamthomasbishop, any opinion?

Reply:
I think em-dash is the way to go, but if a style guide says otherwise let's lean on that.

Reply:
Although actually I guess Android has already merged this with the double dash?

Reply:
No, they used an em-dash – I updated this to match!